cfme/configure/settings.py

```diff
@@ -127,6 +127,10 @@ def step(self):
 
 
 class Visual(Updateable, Navigatable):
+
+    def __init__(self, appliance=None):
```

Comment:
**REQUIRED** off hand i cant see why this `__init__` is needed - my first impression is that its only replicating the one of `Navigatable` please explain why its needed or remove it

Reply:
Ok

Reply:
@RonnyPfannschmidt : I am initialising appliance with help of that and passing it to `Navigatable.__init__`